portfolio/src/main/webapp/index.html

```diff
@@ -72,18 +72,17 @@ <h3>Aspiring Software Engineer</h3>
 		<div class="container">
 			<div class="row">
 				<div class="about">
-					<center>
-					<div class="clearfix"></div>
-					<h2>Courses Taken</h2>
+                    <center>
+					<h2>Courses</h2>
 					<p class="struct">
 						Data Structures and Algorithms (CS 225)<br>
```

Comment:
nit: you could use a <ul><li> (unordered list), which would let you achieve a similar behavior in a slightly cleaner way (no need to specify the line break, for example).

If you don't want to see the bullet point, you can also disable it in the CSS.